### PR TITLE
`stdout-inline` updates

### PR DESCRIPTION
**Pull request by sschneid**
- auto-pass `-s` (`--capture=no`) to PyTest
- spacing/display tweaks

### CODE REVIEW

**Comment by sschneid** [comment]
Test failures unrelated to code (looks like CoreOS was updated in our CF templates), but I will fix-up the failing test and attach it to this PR.

**Comment by sschneid** [comment]
@mesosphere-ci retest this please

File: shakedown/cli/main.py [hunk 1]
```diff
@@ -327,6 +326,9 @@ def pytest_sessionfinish(self, session, exitstatus):
         for opt in args['pytest_option']:
             opts.append(opt)
 
+    if args['stdout_inline']:
+        opts.append('-s')
```
Comment (jeschkies) [inline shakedown/cli/main.py]: Yes. Sweet!

**Comment by kensipe** [comment]
LGTM

**Pull request merged**